packages/datadog-shimmer/index.js

```diff
@@ -0,0 +1,3 @@
+'use strict'
+
```

Comment:
Files like this (i.e. just an immediate require-and-export) should be unnecessary. Is there any way we can avoid it? Maybe with a package.json or something?

Reply:
It will no longer be needed when we switch to actual modules, but in the meantime I think this is cleaner than a random `package.json` that has no use.

Reply:
I mean, this is an `index.js` that has no use. 😄 

Not gonna block on this.